pkg/sql/logictest/testdata/logic_test/interval

```diff
@@ -367,3 +367,12 @@ subtest regression_62369
 
 query error "10000000000000000000000000000000000": value out of range
 SELECT INTERVAL '10000000000000000000000000000000000 year'
+
+query T
+SELECT i / 2 FROM ( VALUES
+  ('0 days 0.253000 seconds'::interval),
+  (INTERVAL '0.000001'::interval)
```

Comment:
can you add cases for 0.000002 all the way up to 0.00001?

Reply:
do you mean the case like '0.000002'::interval?

Reply:
yep!